src/test/java/org/gridsuite/modification/server/LinesAttachToSplitLinesTest.java

```diff
@@ -0,0 +1,255 @@
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentMatchers;
+import org.mockito.stubbing.Answer;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.*;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.web.client.RestTemplate;
+import java.util.*;
+import java.util.stream.Collectors;
+import static org.gridsuite.modification.server.NetworkModificationException.Type.*;
+import static org.junit.Assert.*;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * @author David Braquart <david.braquart at rte-france.com>
+ */
+@RunWith(SpringRunner.class)
+@AutoConfigureMockMvc
+@SpringBootTest
+public class LinesAttachToSplitLinesTest {
+
+    private static final UUID TEST_NETWORK_ID = UUID.fromString("7928181c-7977-4592-ba19-88027e4254e4");
+    private static final UUID TEST_GROUP_ID = UUID.randomUUID();
+    private static final UUID TEST_REPORT_ID = UUID.randomUUID();
+
+    private static final String CREATE_URI_STRING = "/v1/networks/{networkUuid}/lines-attach-to-split-lines?group=" + TEST_GROUP_ID + "&reportUuid=" + TEST_REPORT_ID + "&reporterId=" + UUID.randomUUID();
+    private static final String UPDATE_URI_STRING = "/v1/modifications/%s/lines-attach-to-split-lines-creation";
+    private static final String DELETE_URI_STRING = "/v1/groups/" + TEST_GROUP_ID + "/modifications";
+    private static final String COPY_URI_STRING = "/v1/groups/" + TEST_GROUP_ID + "?action=COPY";
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper mapper;
+    @MockBean
+    private NetworkStoreService networkStoreService;
+
+    @Autowired
+    private NetworkModificationRepository modificationRepository;
+
+    @MockBean
+    @Qualifier("reportServer")
+    private RestTemplate reportServerRest;
+
+    @Autowired
+    private NetworkModificationService networkModificationService;
+
+    private ObjectWriter objectWriter;
+    private Network network;
+
+    @Before
+    public void setUp() {
+        objectWriter = mapper.writer().withDefaultPrettyPrinter();
+        // /!\ create a new network for each invocation (answer)
+        when(networkStoreService.getNetwork(TEST_NETWORK_ID)).then((Answer<Network>) invocation -> {
```

Comment:
Do not create a new network for each invocation

Reply:
Maybe just use @BeforeAll for network creation